File: AndroidBotNavView/app/src/main/java/com/example/shakil/androidbotnavview/MainActivity.java
package com.example.shakil.androidbotnavview;

import androidx.appcompat.app.AppCompatActivity;

import android.graphics.Color;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.Toast;

import com.aurelhubert.ahbottomnavigation.AHBottomNavigation;
import com.aurelhubert.ahbottomnavigation.AHBottomNavigationItem;

public class MainActivity extends AppCompatActivity {

    AHBottomNavigation bottomNavigation;
    Button btn_count;

    int count = 0;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        bottomNavigation = findViewById(R.id.bottom_nav);

        AHBottomNavigationItem item1 = new AHBottomNavigationItem("Tab 1", R.drawable.ic_add_alert_black_24dp, android.R.color.white);
        AHBottomNavigationItem item2 = new AHBottomNavigationItem("Tab 2", R.drawable.ic_add_location_black_24dp, android.R.color.white);
        AHBottomNavigationItem item3 = new AHBottomNavigationItem("Tab 3", R.drawable.ic_add_shopping_cart_black_24dp, android.R.color.white);

        bottomNavigation.addItem(item1);
        bottomNavigation.addItem(item2);
        bottomNavigation.addItem(item3);

        // Set background color
        bottomNavigation.setDefaultBackgroundColor(Color.parseColor("#190019"));

        btn_count = findViewById(R.id.btn_count);
        btn_count.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                count++;
                bottomNavigation.setNotification("" + count, 0);
            }
        });

        bottomNavigation.setOnTabSelectedListener(new AHBottomNavigation.OnTabSelectedListener() {
            @Override
            public boolean onTabSelected(int position, boolean wasSelected) {
                Toast.makeText(MainActivity.this, "Start Activity " + position, Toast.LENGTH_SHORT).show();
                return true;
            }
        });
    }
}

File: KotlinBotNavView/app/src/main/java/com/example/shakil/kotlinbotnavview/MainActivity.kt
package com.example.shakil.kotlinbotnavview

import android.graphics.Color
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Toast
import com.aurelhubert.ahbottomnavigation.AHBottomNavigationItem
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    var count:Int = 0

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val item1 = AHBottomNavigationItem("Tab 1", R.drawable.ic_add_alert_black_24dp, android.R.color.white)
        val item2 = AHBottomNavigationItem("Tab 2", R.drawable.ic_add_location_black_24dp, android.R.color.white)
        val item3 = AHBottomNavigationItem("Tab 3", R.drawable.ic_add_shopping_cart_black_24dp, android.R.color.white)

        bottom_nav.addItem(item1)
        bottom_nav.addItem(item2)
        bottom_nav.addItem(item3)

        // Set background color
        bottom_nav.setDefaultBackgroundColor(Color.parseColor("#190019"))

        btn_count.setOnClickListener {
            count++
            bottom_nav.setNotification(count.toString(), 0)
        }

        bottom_nav.setOnTabSelectedListener { position, wasSelected ->
            Toast.makeText(this@MainActivity, "Start Activity: " + position, Toast.LENGTH_SHORT).show()
            true
        }
    }
}
